rainbow-me/rainbowkit
# feat: `bloomWallet`

## Changes
- Added `bloomWallet`

## What to test
1. Install Bloom Wallet on desktop [here](https://bloomwallet.io/)
2. Once installed open the desktop app and create a wallet
3. Go to RainbowKit modal and make sure the wallet connection works smoothly
